packages/multicast_dns/CHANGELOG.md

```diff
@@ -2,4 +2,8 @@
 
 * Initial Open Source release.
 * Migrates the dartino-sdk's mDNS client to Dart 2.0 and Flutter's analysis rules
-* Breaks from original Dartino code, as it does not use native libraries for macOS and overhauls the `ResourceRecord` class.
\ No newline at end of file
+* Breaks from original Dartino code, as it does not use native libraries for macOS and overhauls the `ResourceRecord` class.
+
+## 0.1.1
```

Comment:
Nit: we usually put these in descending order (so this should be on top) so the user sees the newest things first.

Reply:
makes sense!